Shared/View/BaseItemListView.swift

```diff
@@ -160,6 +160,10 @@ extension BaseItemListView {
 
                         cell.titleLabel.text = title
                         cell.detailLabel.text = username
+                        
+                        if (self.extensionContext == nil) {
+                            cell.accessoryType = .disclosureIndicator
+                        }
```

Comment:
This doesn't drive me wild with approval. I'd prefer to use a method that gets overridden in the subclass (removes the check for `extensionContext` but adds a method call).

However, given timescales, I think I'd label this as #technical-debt, approve it and move on.

Reply:
Agreed on both points. I'll file a tech-debt story to come back to this since I'm not clever enough to successfully define methods myself (yet). 😔